doc/hop.txt

```diff
@@ -416,6 +416,29 @@ below.
     Defaults:~
         `keys = 'asdghklqwertyuiopzxcvbnmfj'`
 
+`quit_key`                                                   *hop-config-quit_key*
+    A string representing a key that will quit hop mode without also feeding
+    that key into neovim to be treated as a normal key press.
+
+    It is possible to quit hopping by pressing any key that is not present in
+    |hop-config-keys| however when you do this the keys normal function is
+    also performed. For example if hopping in |visual-mode|, pressing <Esc> will
+    quit hopping and also exit |visual-mode|.
+
+    If the user presses `quit_key`, hop will be quit without the keys normal
+    function being performed. For example if hopping in |visual-mode| with
+    `quit_key` set to '<Esc>', pressing <Esc> will quit hoppping without
+    quitting |visual-mode|.
+
+    Note:~
+        `quit_key` should only contain a single key.
+    Note:~
+        `quit_key` should not contain a key that is also present in
+        |hop-config-keys|.
+
+    Defaults:~
+        `quit_key = ''`
```

Comment:
What does that mean? I think we should set it to `<Esc>` by default.

Reply:
Ok I'll change the default to `<Esc>`.